### PR TITLE
Blockchaincom withdrawal example and add withdrawal arguement guard

### PR DESCRIPTION
**Pull request by DoctorSlimm**
Add Blockchaincom-specific withdrawal example, and add argument guard in blockchaincom.js for address argument.

### CODE REVIEW

File: examples/js/blockchaincom-withdrawal.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+const exchange = new ccxt.blockchaincom({
+    'secret': 'SECRET',
+})
+
+// blockchaincom specific internal beneficiary id
+const withdrawal_beneficiary = 'WITHDRAWAL_BENEFICIARY'
+
+
+
+require('ansicolor').nice
```
Comment (kroitor) [inline examples/js/blockchaincom-withdrawal.js:17]: @DoctorSlimm thx! Would be great if you could clean out the requirement for `ansicolor` from it.

Comment (DoctorSlimm) [reply examples/js/blockchaincom-withdrawal.js:17]: Sure thing

File: js/blockchaincom.js [hunk 2]
```diff
@@ -930,14 +930,18 @@ module.exports = class blockchaincom extends Exchange {
          * @param {string} address the address to withdraw to
          * @param {string|undefined} tag
          * @param {object} params extra parameters specific to the blockchaincom api endpoint
+         * @param {string} params.beneficiary blockchaincom specific id of the whitelisted beneficiary, can be retrieved by calling fetchWithdrawalWhitelist()
          * @returns {object} a [transaction structure]{@link https://docs.ccxt.com/en/latest/manual.html#transaction-structure}
          */
         await this.loadMarkets ();
+        if (address !== undefined) {
+            throw new ExchangeError (this.id + ' withdraw () does not support the address argument, provide beneficiary parameter in params instead, for a whitelisted withdrawal address. These can be found by calling fetchWithdrawalWhitelist ()');
```
Comment (kroitor) [inline js/blockchaincom.js:938]: Actually, we can support specifying the beneficiary in the address argument in this case? @DoctorSlimm lmk if you can revisit this once more.

Comment (DoctorSlimm) [reply js/blockchaincom.js:938]: Sure thing will do

**Pull request closed without merging**